src/main/java/uk/gov/hmcts/reform/sscs/service/FuzzyMatcherService.java

```diff
@@ -212,6 +213,7 @@ private void logMessage(String caseId, String code, BoundExtractedResult<Pair<St
 
     private static Set<Pair<String, Benefit>> getBenefitShortNameAndDescriptionFuzzyChoices() {
         return stream(values())
+            .filter(e -> e.getSscsType().equals(SscsType.SSCS1))
```

Comment:
This is filtering out sscs2 as well which wasn't before. Is that what expected?

Reply:
Yes because SSCS2 doesn't have a space on the form to write the benefit type - it defaults to 'Child Support'